roles/confluent.common/tasks/main.yml

```diff
@@ -1,13 +1,12 @@
 ---
 # JVM Installation and Repo Setup
-- include_tasks: redhat.yml
-  when: ansible_os_family == "RedHat"
+- include_tasks: "{{ ansible_os_family }}.yml"
```

Comment:
I like what you did here, but I think ansible_os_family resolves to RedHat and Debian not RedHat, Ubuntu, and Debian
Meaning the Ubuntu tasks will not be run

Im curious what ansible_distribution resolves to on a redhat/centos host, maybe that would work

Reply:
I might have a solution - pushing it now

Reply:
> Hello,
> 
> roles/confluent.common/tasks/main.yml
> 
> Dynamic inclusion of ansible_os_family.yaml tasks are not working and breaking the playbook. Tested it on Ubuntu and its not including the Ubuntu.yml

Thx - fair point. I implemented a different approach now